### ISSUE TITLE
dockerAll is not returning expected data in latest version 

### ISSUE THREAD

**Issue by superhero** [sebhildebrandt/systeminformation#244]
**Describe the bug**
dockerAll is not returning expected data in latest version 

**To Reproduce**
Use latest version 4.11.4 of `systeminformation` instead of version 3.54.0
Use endpoint `dockerAll` 
Get unexpected `undefined` values

**Current Output**
```
   id: '92ba2c1abe08e06466c1af87c825ee2f7decabaa6752b84acc81939b6265ba10',
  name: 'fervent_bassi',
  image: 'hello-world',
  imageID: 'sha256:fce289e99eb9bca977dae136fbe2a82b6b7d4c372474c9235adc1741675f587e',
  command: '/hello',
  created: 1560866512,
  started: 1560866513,
  finished: 1560866513,
  createdAt: '2019-06-18T14:01:52.962035775Z',
  startedAt: '2019-06-18T14:01:53.29490287Z',
  finishedAt: '2019-06-18T14:01:53.332951027Z',
  state: 'exited',
  restartCount: 0,
  platform: '',
  driver: 'aufs',
  ports: [],
  mounts: [],
  mem_usage: undefined,
  mem_limit: undefined,
  mem_percent: undefined,
  cpu_percent: undefined,
  pids: undefined,
  netIO: undefined,
  blockIO: undefined,
  cpu_stats: undefined,
  precpu_stats: undefined,
  memory_stats: undefined,
  networks: undefined,
  processes: []
```

**Expected behavior**
```
    id: '92ba2c1abe08e06466c1af87c825ee2f7decabaa6752b84acc81939b6265ba10',
    name: 'fervent_bassi',
    image: 'hello-world',
    imageID: 'sha256:fce289e99eb9bca977dae136fbe2a82b6b7d4c372474c9235adc1741675f587e',
    command: '/hello',
    created: 1560866512,
    state: 'exited',
    ports: [],
    mounts: [],
    mem_usage: 0,
    mem_limit: 0,
    mem_percent: 0,
    cpu_percent: 0,
    pids: 0,
    netIO: { rx: 0, wx: 0 },
    blockIO: { r: 0, w: 0 },
    cpu_stats: { cpu_usage: [Object], throttling_data: [Object] },
    precpu_stats: { cpu_usage: [Object], throttling_data: [Object] },
    memory_stats: {},
    networks: {},
    processes: []
```

**Environment (please complete the following information):**
 - OS: `Linux work 3.16.0-5-amd64 #1 SMP Debian 3.16.51-3+deb8u1 (2018-01-08) x86_64 GNU/Linux`
 - Hardware

```
Architecture:          x86_64
CPU op-mode(s):        32-bit, 64-bit
Byte Order:            Little Endian
CPU(s):                4
On-line CPU(s) list:   0-3
Thread(s) per core:    1
Core(s) per socket:    4
Socket(s):             1
NUMA node(s):          1
Vendor ID:             GenuineIntel
CPU family:            6
Model:                 60
Model name:            Intel(R) Core(TM) i5-4670 CPU @ 3.40GHz
Stepping:              3
CPU MHz:               3460.960
CPU max MHz:           3800.0000
CPU min MHz:           800.0000
BogoMIPS:              6784.64
Virtualization:        VT-x
L1d cache:             32K
L1i cache:             32K
L2 cache:              256K
L3 cache:              6144K
NUMA node0 CPU(s):     0-3
```

**Comment by sebhildebrandt** [comment]
@superhero 
OK, I will try to fix that as soon as possible.

**Comment by sebhildebrandt** [comment]
@superhero : fixed! Version 4.11.5 just released
